packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -97,7 +97,7 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
       if (!objectMetadata) {
         throw new FieldMetadataException(
           'Object metadata does not exist',
-          FieldMetadataExceptionCode.OBJECT_METADATA_NOT_FOUND,
+          FieldMetadataExceptionCode.INVALID_FIELD_INPUT,
```

Comment:
What's the motivation behind this change? Should we do it for update and delete too?

Reply:
Oh no ok I see, one throws a 4XX and the other throws a 500. Given this comes from the input and not the update/delete, that makes sense. LGTM 👍